src/Ydb.Sdk/src/Services/Auth/Login.cs

```diff
@@ -0,0 +1,72 @@
+using Ydb.Auth;
+using Ydb.Auth.V1;
+using Ydb.Sdk.Client;
+
+namespace Ydb.Sdk.Services.Auth;
+
+public class LoginSettings : OperationRequestSettings
+{
+}
+
+public class LoginResponse : ResponseWithResultBase<LoginResponse.ResultData>
+{
+    internal LoginResponse(Status status, ResultData? result = null)
+        : base(status, result)
+    {
+    }
+
+    public class ResultData
+    {
+        public string Token { get; }
+
+        internal ResultData(string token)
+        {
+            Token = token;
+        }
+
+
+        internal static ResultData FromProto(LoginResult resultProto)
+        {
+            var token = resultProto.Token;
+            return new ResultData(token);
+        }
+    }
+}
+
+public partial class AuthClient
```

Comment:
Why you need split the class for two files?
It is difficult see and understand full implementation of the class.

What about implement the class into AuthClient.cs?